internal/pkg/install/install.go

```diff
@@ -120,9 +131,88 @@ func Install(data *userdata.UserData) (err error) {
 			}
 		}
 	}
+
+	if err = writeExtLinux(); err != nil {
+		return err
+	}
+	if err := unix.Unmount("/root/boot", 0); err != nil {
+		return err
+	}
+	if err := unix.Unmount("/root/var", 0); err != nil {
+		return err
+	}
+	if err := unix.Unmount("/root", 0); err != nil {
+		return err
+	}
+	log.Println("writing gptmbr.bin to the block device")
+	b, err := ioutil.ReadFile("/usr/share/gptmbr.bin")
+	if err != nil {
+		return err
+	}
+	f, err := os.OpenFile("/dev/sda", os.O_WRONLY, os.ModeDevice)
+	if err != nil {
+		return err
+	}
+	if _, err := f.Write(b); err != nil {
+		return err
+	}
+	f.Close()
+	log.Println("rebooting")
+	time.Sleep(5 * time.Second)
+	unix.Reboot(int(unix.LINUX_REBOOT_CMD_RESTART))
```

Comment:
I'm torn on the reboot here. I can see why, but I'd also argue it's unnecessary since after we get done with the install, init does the switchroot and at that point it should be no different if it were booted.

Reply:
As long as we're fully testing that the reboot works and we have no reason to think it would break for users, it sounds pretty nice to not have machines rebooting.

Reply:
We should be careful of this as uptime can trend toward very large numbers, and not being able to reboot would be a nasty surprise in the event of an outage.

Reply:
Agree. This is mostly here for the purpose of testing. I imagine that e2e tests should test if a node survives a reboot.